css/styles.css

```diff
@@ -0,0 +1,3 @@
+.body {
+    background: #FFF;
+}
```

Comment:
exist only one tag body that contain all others tags in it and all styles that must be included into body are written just like:
body {
// properties
}

it the basic practice to write all in TAG body, don't create unnecessary CLASS body

Reply:
also add in styles.css default styles to tags:

main {
  overflow-x: hidden;
}

:active,
:hover,
:focus {
  outline: 0;
  outline-offset: 0;
  transition: all var(--main-transition);
}

li {
  list-style: none;
}

a {
  text-decoration: none;
  margin: 0;
  padding: 0;
}

a, button {
  cursor: pointer;
}

Reply:
added those properties

Reply:
where? I don't see it in files changed